napari_aicssegmentation/controller/workflow_steps_controller.py

```diff
@@ -30,3 +30,7 @@ def index(self):
     def close_workflow(self):
         self.model.reset()
         self.router.workflow_selection()
+
+    def run_all_and_add_image(self):
```

Comment:
1. This method needs to be added to the controller's interface - for code completion purposes
2. nit: I would simplify the name of this method (maybe just `run_all` or `run_all_steps`). The "add image" part is sort of an implementation detail, or rather it's really the result of the method.